packages/expo-web-browser/android/src/test/java/expo/modules/webbrowser/TestUtils.kt

```diff
@@ -45,18 +42,13 @@ fun browserArguments(
   enableDefaultShareMenuItem: Boolean = true,
   showInRecents: Boolean = true,
   createTask: Boolean = true
-): ReadableArguments {
-  // Move creation of readable arguments to TestUtils
-  return readableArgumentsOf(
-    mapOf(
-      "toolbarColor" to toolbarColor,
-      "toolbarSecondaryColor" to toolbarSecondaryColor,
-      "browserPackage" to browserPackage,
-      "enableBarCollapsing" to enableBarCollapsing,
-      "showTitle" to showTitle,
-      "enableDefaultShareMenuItem" to enableDefaultShareMenuItem,
-      "showInRecents" to showInRecents,
-      "createTask" to createTask
-    )
-  )
+) = OpenBrowserOptions().apply {
+  this.toolbarColor = toolbarColor
+  this.secondaryToolbarColor = toolbarSecondaryColor
+  this.browserPackage = browserPackage
+  this.enableBarCollapsing = enableBarCollapsing
+  this.showTitle = showTitle
+  this.enableDefaultShareMenuItem = enableDefaultShareMenuItem
+  this.showInRecents = showInRecents
+  this.shouldCreateTask = createTask
```

Comment:
Hmm, what if we convert the `OpenBrowserOptions` to the data class and use the contractor with named arguments instead of this apply block. I think it may be more readable.

Reply:
Do records work with data classes?

Reply:
Yep, it should work (if I remember correctly) ;)